Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Reflection.PortableExecutable;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static float globalHeight = 0f;
+        public static float globalWeight = 0f;
+        public static string bmiError = "Failed calculating BMI. Reason:";
+
+        public static string Information()
+        {
+            Console.WriteLine("Testing");
+            string firstName = Console.ReadLine();
+            string errorMsg = "Name cannot be empty.";
+
+            if (String.IsNullOrEmpty(firstName))
+            {
+                Console.WriteLine(errorMsg);
+                return errorMsg;
+            }
+            else
+            {
+                return firstName;
+            }
+        }
+
+        public static int AskAge()
+        {
+            Console.WriteLine("Testing");
+            string sAge = Console.ReadLine();
+
+            int age;
+            bool isNumber = int.TryParse(sAge, out age);
+
+            if (!isNumber) return -1;
+            if (age == 0) return 0;
+            if (age >= 0) return age;
+
+            return 0;
+
+        }
+
+        public static float Weight()
```

Comment:
Keep it consistent. AskAge was a good name. AskWeight, AskHeight should be how you name the other methods too, because that's what they do.